source/MultiGrid/MultiGrid.js

```diff
@@ -611,32 +615,54 @@ class MultiGrid extends React.PureComponent {
       fixedColumnCount,
       fixedRowCount,
       rowCount,
+      scrollTop,
+      hideBottomLeftGridScrollbar,
     } = props;
     const {showVerticalScrollbar} = this.state;
 
     if (!fixedColumnCount) {
       return null;
     }
 
-    const additionalRowCount = showVerticalScrollbar ? 1 : 0;
+    const additionalRowCount = showVerticalScrollbar ? 1 : 0,
+      height = this._getBottomGridHeight(props),
+      width = this._getLeftGridWidth(props),
+      scrollbarSize = this.state.showVerticalScrollbar ? this.state.scrollbarSize : 0,
```

Comment:
`scrollbarSize` is now in `state.instanceProps`.

Reply:
I think state structure change in Grid but not in MutliGrid so it is working fine. 
Also, I ran `yarn lint` and it seem to be all good too.

So we are good here?